rllib/algorithms/ddpg/ddpg_torch_policy.py

```diff
@@ -266,7 +266,9 @@ def loss(
             q_tp1 = torch.min(q_tp1, twin_q_tp1)
 
         q_tp1_best = torch.squeeze(input=q_tp1, axis=len(q_tp1.shape) - 1)
-        q_tp1_best_masked = (1.0 - train_batch[SampleBatch.DONES].float()) * q_tp1_best
+        q_tp1_best_masked = (
```

Comment:
Note on everything below: In our loss functions, `DONES` got replaced by `TERMINATEDS` in 99% of the cases. This yields more accuracy now as e.g. Q-val=0.0 should only happen if the episode is really terminated, and not when it's just truncated.